common/static/common/js/discussion/views/new_post_view.js

```diff
@@ -100,6 +109,26 @@
                 }
             };
 
+            NewPostView.prototype.getGroupName = function() {
+                var userGroupId;
+                var cohort;
+                var group_name = null;
+                if (this.course_settings.get('is_cohorted')) {
+                    userGroupId = $('#discussion-container').data('user-group-id');
+                    if (!userGroupId) {
+                        userGroupId = this.user_group_id;
+                    }
+                    cohort = _.find(this.course_settings.get('cohorts'), function(cohort) {
```

Comment:
Nit: I'm not sure if the es2015 transpiler is activated yet on all .js files in the platform, but if it is then we could just use `this.course_settings.get('cohorts').find(cohort => cohort.id === userGroupId)`. (i.e. replace underscore with the standard JS method).